### PR TITLE
Updated sample apps

### PR DESCRIPTION
**Pull request by nschapeler**
Updated encrypt-, hash-, signing-, and TUISample and Boilerplate.

Updated to RivetzJ 1.0.12.
Updated to the latest android gradle plugin.
Fixed gradle script issues.
Enabled all lint warnings.
Fixed all lint warnings.
Made lint warnings fatal.
Replaced rivetzwalletactivity with rivetzapiactivity
Added the new legal disclaimer at the top

I still need to update the activity on the apps & the last commit should say "updated TUIsample" instead of "updated hashsample"

### CODE REVIEW

File: RivetzBoilerPlate/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">RivetzBoilerplate</string>
+    <string name="button">RivetzBoilerplate</string>
```
Comment (nschapeler) [inline RivetzBoilerPlate/app/src/main/res/values/strings.xml]: Change the strings here

File: TUISample/app/build.gradle [hunk 2]
```diff
@@ -2,38 +2,44 @@ apply plugin: 'com.android.application'
...
+            minifyEnabled false
         }
-
     }
 
-
-
     tasks.withType(JavaCompile) {
-
         options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
```
Comment (nschapeler) [inline TUISample/app/build.gradle]: Delete the whole task here

File: EncryptDecryptSample/app/src/androidTest/java/com/rivetz/encryptdecryptexample/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.rivetz.r_app2;
```
Comment (kitfitz) [inline EncryptDecryptSample/app/src/androidTest/java/com/rivetz/encryptdecryptexample/ExampleInstrumentedTest.java]: The package name must match the directory structure. It looks like you updated the package name, but need to move the file(s).

**Comment by nschapeler** [comment]
Fixed the naming issues, all paths and packages are called encryptdecrypt sample now aside from the directory of mainactivity. However when I look locally & on GitHub, it says encryptdecryptSAmple, as it should.

**Pull request merged**